src/content.ts

```diff
@@ -0,0 +1,72 @@
+const fetchData = () => {
+  const emoteData = [];
+  fetch('https://api.betterttv.net/3/cached/emotes/global', {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+    .then((response) => response.json())
+    .then((json) => {
+      json.forEach((x) => {
```

Comment:
```suggestion
      json.forEach(({id, code}) => {
```
this is called [object desctructuring](https://basarat.gitbook.io/typescript/future-javascript/destructuring#object-destructuring) which is super useful for a case like this where we are solely conserned about the _attributes_ of x, not x itself. 

You could then say `const tempObj =  { name: code, id }` or better yet, emoteData.push({ name: code, id })` and remove the need for declaring a variable.

Reply:
It would also help if `x` was more descriptive, maybe name it `emote`